src/main/java/org/betonquest/betonquest/conversation/SlowTellrawConvIO.java

```diff
@@ -39,16 +40,15 @@ public void display() {
         }
 
         // NPC Text
-        final List<String> lines = new ArrayList<>(Arrays.asList(LocalChatPaginator.wordWrap(
-                Utils.replaceReset(textFormat.replace("%npc%", npcName) + npcText, npcTextColor),
-                60)));
-
+        final String[] lines = LocalChatPaginator.wordWrap(
+                Utils.replaceReset(textFormat.replace("%npc%", npcName) + npcText, npcTextColor), 50);
+        final AtomicInteger lineCount = new AtomicInteger(0);
         endLines = new ArrayList<>();
 
         new BukkitRunnable() {
             @Override
             public void run() {
-                if (lines.isEmpty()) {
+                if (lineCount.get() == lines.length) {
```

Comment:
Why no just move the modifiable variable for line count into the method body? No need to use a AtomicInteger.